### ISSUE TITLE
[Bug] Risk thresholds for metrics configurable fr frequentist stats engine

### ISSUE THREAD

**Issue by bosyk** [growthbook/growthbook#1214]
### Summary

When using the frequentist stats engine, I can change bayesian properties for a metric.

#### Expected Behavior

When using the frequentist stats engine, I can only change behaviour relevant for frequentist statistics. An example would be the section from v1.9.0

<img width="414" alt="image" src="https://user-images.githubusercontent.com/30110352/233641297-0668ccdf-56fd-47b0-abec-ce8ab2e6780b.png">

#### Current Behavior

When using the frequentist stats engine, I can change behaviour relevant for bayesian statistics. An example would be the section from v2.0.0

<img width="413" alt="image" src="https://user-images.githubusercontent.com/30110352/233641487-b73bfb00-4a8b-4662-a01e-25c351c9fc7d.png">

**Comment by lukesonnet** [comment]
Thanks for opening this issue. The reason this is currently still an option is that we envision the possibility of being able to see results in both engines within the same experience and letting you toggle between the two on the fly. Allowing this would mean we should also allow people to configure settings that apply to one engine or the other.

That said, we should figure out a way to improve the UX so that users who only want one engine or the other don't see settings that don't apply to them.

As such, I'm not sure I would describe this as a bug, but I agree the UX is sub-optimal.

**Comment by bosyk** [comment]
I agree, it's not a bug, but an unexpected ed change in the UI's behaviour. I'm looking forward to the features and improvements you teasered 😉
